README.md

```diff
@@ -1,40 +1,131 @@
 
-3.Clone the repository and navigate into it
+**Note:** Currently only Yarn understands the [workspace](https://yarnpkg.com/lang/en/docs/workspaces/) package setting used in the OpenUI5 repository. If you do not plan to work with OpenUI5 you might as well use npm. But keep in mind that linking the same module with npm and Yarn might lead to issues. Also, Yarn can't work with links created by npm and vice versa. See [FAQ: What's the thing with yarn?](https://github.com/SAP/ui5-tooling#whats-the-thing-with-yarn) for details.
 
-```sh
-git clone https://github.com/SAP/openui5-basic-template-app.git
-cd openui5-basic-template-app
-```
+### Prerequisites
 
-4.Install all npm dependencies
+- [Yarn](https://yarnpkg.com/en/docs/install) (**version must be 1.0 or higher**)
+    - *Note that you can use npm instead of Yarn if you do not plan to work with a local copy of the [OpenUI5 main repository](https://github.com/SAP/openui5). See [FAQ: What's the thing with yarn?](https://github.com/SAP/ui5-tooling#whats-the-thing-with-yarn)*
 
-```sh
-npm install
-```
+### Preparation
+The following needs to be done just once per setup.
 
-5.Install all bower dependencies
+1. Clone the OpenUI5 repository and navigate into it
+    **Note:** The UI5 version must be 1.52.5 or higher, you can check that in the root `package.json` file
+    ```sh
+    git clone https://github.com/SAP/openui5.git
+    cd openui5
+    ```
+1. Install all dependencies (this also links all OpenUI5 libraries between each other)
+    ```sh
+    yarn
+    ```
+1. Make all projects available as global links. **Note**: The OpenUI5 project uses [wsrun](https://github.com/whoeverest/wsrun) to link all libraries with one command. See [Linking Projects](https://github.com/SAP/ui5-tooling#linking-projects) for general information about project linking.
+    ```sh
+    yarn run link-all
+    ```
 
-```sh
-bower install
-```
+### Linking
+1. In your application directory: Link the required OpenUI5 libraries
+    ```sh
+    yarn link @openui5/sap.ui.core
+    yarn link @openui5/sap.m
+    yarn link @openui5/sap.ui.layout
+    yarn link @openui5/themelib_sap_belize
+    ```
 
-6.Run grunt to lint, build and run a local server (have a look into `Gruntfile.js` to see all the tasks).
+You can now make changes in your local OpenUI5 repository and see the impact directly when serving or building your application.
 
-```sh
-grunt
-```
+### Unlinking
+To return to using the OpenUI5 npm packages
 
-7.Open the app in your browser: [http://localhost:8080](http://localhost:8080)
+1. Remove the links
+    ```sh
+    yarn unlink @openui5/sap.ui.core
+    yarn unlink @openui5/sap.m
+    yarn unlink @openui5/sap.ui.layout
+    yarn unlink @openui5/themelib_sap_belize
+    ```
+1. Re-install the packages from the registry
+    ```sh
+    yarn
+    ```
```

Comment:
In the end of all 3 readme's  please add a new section at the end:

## Support
This repository is based on the [OpenUI5 template demo apps](https://openui5.hana.ondemand.com/demoapps.html) and updated regularly with our latest recommendations. If you found a bug, please create an [OpenUI5 issue](https://github.com/sap/openui5/issues). Thank you!